syntax = "proto3";

package ChatMessages;

message UserMessage {
	uint64 id = 1;
	string name = 2;
	string ipAddress = 3;
	string timestamp = 4;
	string data = 5;
}